drivers/bch/bchdev_driver.c

```diff
@@ -459,6 +459,13 @@ static int bch_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
           if (bchinode->u.i_bops->ioctl != NULL)
             {
               ret = bchinode->u.i_bops->ioctl(bchinode, cmd, arg);
+
+              /* Drivers may not support command BIOC_FLUSH */
+
+              if (ret == -ENOTTY && cmd == BIOC_FLUSH)
```

Comment:
yes, but it could be safer to pass ioctl to driver

Reply:
Just checking, this works for me, so it is fine by me! I was only asking, because I had fixed the same thing in a bit different way (https://github.com/apache/nuttx/pull/15259)